assets/models/system/Mistral-7B-v0-1/spec.yaml

```diff
@@ -0,0 +1,53 @@
+$schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
+name: Mistral-7B-Instruct-v0-1
```

Comment:
Mistral-7B-Instruct-v01

Reply:
updated to `Mistral-7B-v01` for base model